### DISCUSSION TITLE
Anybody with a Pugio II mouse? Question before buying

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#2821] [Q&A]
Hi all! Does anybody have a ROG Pugio II mouse?
I'm thinking about getting one because Razer can't do wireless, but I'd like to ask you if you can configure the four side buttons with G-helper. And if you can, do the bindings get stored in the internal memory of the mouse?

Thank you very much!

**Comment by seerge** [comment]
@LoreAPC hello,

ROG Pugio II should be supported. You can't rebind buttons using G-Helper atm (for any mouse). But you can try to do that using Sharpkeys or Autohotkey assuming buttons emit standard keycodes for for 3rd, 4th, etc mouse buttons.

**Comment by LoreAPC** [comment]
Thanks, I didn't know that rebinding isn't implemented...since I never owned a ROG peripherial I sorta ignored this feature in g-helper and its development.
My usecase is a bit specific tho: I need to bind the side buttons to ctrl+c and ctrl+v, and also I need this binding to be functional when using the mouse on another pc via bluetooth without any software.
I don't want to install AC only for this, I guess it's not the right device for me...

**Reply by seerge** [reply]
@LoreAPC i think you can bind it to Ctrl + C / Ctrl + V even in software-way (via sharpkeys or autohotkey). But you will need that software to be running on each device you use the mouse with  :)

**Comment by LoreAPC** [comment]
Wait a minute there, I just discovered about ROG Armoury II. Is it a mouse-only software?

**Reply by seerge** [reply]
@LoreAPC yes, but as I know it supports only a few newest peripherals.
